Rakefile

```diff
@@ -1,6 +1,6 @@
 # Add your own tasks in files placed in lib/tasks ending in .rake,
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
-require File.expand_path('../config/application', __FILE__)
+require File.expand_path('config/application', __dir__)
```

Comment:
Is this required?

Reply:
Yes, it was an issue that was raised by govuk-lint-ruby. Although not strictly related to our work we did.